catroid/src/main/java/org/catrobat/catroid/content/bricks/SetLookBrick.java

```diff
@@ -182,18 +182,23 @@ public Brick clone() {
 
 	@Override
 	public List<SequenceAction> addActionToSequence(Sprite sprite, SequenceAction sequence) {
-		sequence.addAction(sprite.getActionFactory().createSetLookAction(sprite, look, wait));
+		if (this instanceof SetBackgroundBrick) {
+			Sprite backgroundSprite = getSpriteToChange();
+			sequence.addAction(sprite.getActionFactory().createSetLookAction(backgroundSprite, look, wait));
+		} else {
+			sequence.addAction(sprite.getActionFactory().createSetLookAction(sprite, look, wait));
+		}
```

Comment:
you are right, for the action it is necessary to use the "SceneToPlay", whereas the Spinners in the IDE need the "CurrentScene".

If anybody has time for writing a test she is welcome to do so.
My previous hotfixes were only on a voluntary basis to help out for the next release :grinning: